core/embed/bootloader/main.c

```diff
@@ -425,6 +425,10 @@ int bootloader_main(void) {
 
     ui_set_initial_setup(true);
 
+#ifndef USE_BACKLIGHT
+    hal_delay(2000);
```

Comment:
is this actually necessary @Hannsek ?
iiuc, this is just prolonging the time the "empty logo" is shown. If we want to prolong time of something, it should be the next `ui_screen_welcome_model` (saying "Trezor Model X") -- currently shown for 1000ms before going to "go to tio/start"

In my opinion, the role of the "empty lock" is to indicate that the device is powered up and didn't get to do anything useful yet. On TT this covers the 400ms needed to initialize touch; there is nothing like that on TR so I would expect the empty lock to basically immediately go away, no?

Reply:
I (hope that I) remember we had the discussion about this with @TychoVrahe; that there was some other reason.

But if there is no other reason, let's do it the same as in TT as lower time can feel buggy.

Reply:
removed. there is no blinking visible, visually it goes straight to the welcome screen. 
https://github.com/trezor/trezor-firmware/commit/86a5251214209609bee1cf3f63b02ac96a18f075

Reply:
added the 1500ms delay for full logo https://github.com/trezor/trezor-firmware/commit/ad4ea54159417c747899734f51cbd556d57b1a3d